setup

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# -e (-o errexit): exit immediately on error
+# -u (-o nounset): treat unset variables as erros
+# -o pipefail: propagate intermediate errors in pipelines
+set -euo pipefail
+
+VAULT_PASSWORD_FILE=~/.forem_selfhost_ansible_vault_password
```

Comment:
@jdoss General question: I'm personally not a big fan of tools that put stuff into the root of my home directory. How do you feel about putting this in `$XDG_CONFIG_HOME/forem/selfhost_ansible_vault_password`. We'd default `$XDG_CONFIG_HOME` to `~/.config` as is the standard.

Reply:
Moving this to `$XDG_CONFIG_HOME/forem/selfhost_ansible_vault_password` sounds great.

Reply:
@jdoss I changed this now. Though to be fair I have hardcoded the vault password location to `~/.config/forem/selfhost_ansible_vault_password` because we also access this value in `ansible.cfg` and I can't be very dynamic there.

Reply:
Cools. That should be fine.